lib/ansible/galaxy/collection/__init__.py

```diff
@@ -1073,14 +1075,15 @@ def _build_files_manifest(b_collection_path, namespace, name, ignore_patterns, m
             namespace,
             name,
             manifest_control,
+            license_file
```

Comment:
Keep the trailing comma?
```suggestion
            license_file,
```

Reply:
Fixed in https://github.com/ansible/ansible/pull/79420/commits/44bbf1b7f9fcc32dde69967a17ae9daa3eae1051.